src/components/UserCover.js

```diff
@@ -6,11 +6,11 @@ import { globalStyleVariables } from '../styles';
 
 const styles = StyleSheet.create({
   avatarContainer: {
-    height: globalStyleVariables.DRAWER_WIDTH * 9 / 16,
+    height: 172,
```

Comment:
what is the reason to fix this value here?

Reply:
The height here  is too high for the landscape layout on iPad.

Reply:
The codebase currently only support for portrait mode. 
In app change orientation is not properly supported yet, for example, when you change orientation in app, the width of illust items are not adjust to fully fit for the window width, some code changes need to be done to proper support landscape mode.
For now, you can revert this height value then I will merge this PR.

Reply:
Well, in my opinion, the landscape layout is necessary, especially on the iPad.
For now I will remove that, and then find something we can do to deal with the rotate bug...

Reply:
Done.